docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
   docker pull nginxinc/nginx-unprivileged:1.23.3
   ```
 
-- Login to your target container registry, as per the following example that uses Google Container Registry (GCR):
+- Login to your target container registry, as per the following example that uses Google Artifact Registry:
+
+   **Note**: The "Service Account" and "Location" used in the below example have to be replaced as applicable.
 
   ```sh
-  docker login -u _json_key --password-stdin https://gcr.io < .ssh/my-service-account-key.json
+  gcloud auth activate-service-account yugabytedb-test@yugabytedb-test-384308.iam.gserviceaccount.com --key-file=key.json
+  gcloud auth configure-docker us-central1-docker.pkg.dev
+  
   ```
 
-- Tag the local images to your target registry, as follows:
+- Tag the local images to your target registry, as follows (Here format for GCP Artifact Registry ):
+
+  **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.
 
   ```sh
-  docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} gcr.io/dataengineeringdemos/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}
-  docker tag nginxinc/nginx-unprivileged:1.23.3 gcr.io/dataengineeringdemos/nginxinc/nginx-unprivileged:1.23.3
-  docker tag postgres:14.4 gcr.io/dataengineeringdemos/postgres:14.4
-  docker tag tianon/postgres-upgrade:11-to-14 gcr.io/dataengineeringdemos/tianon/postgres-upgrade:11-to-14
-  docker tag prom/prometheus:v2.41.0 gcr.io/dataengineeringdemos/prom/prometheus:v2.41.0
+  docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}
```

Comment:
@rajeev-asthana : can you clarify why we want to change these links from gcr.io to us-central1.docker.pkg.dev?

Reply:
@iSignal :   GCR  i.e. Google container registry is deprecated now and is replaced by Google Artifact Registry. Example here is using GCR which needs to be updated with Artifact registry example. "us-central1.docker.pkg.dev" is the syntax for pushing images to Artifact registry in GCP.